plugins/jobs/web_client/templates/JobsGraphWidget.pug

```diff
@@ -0,0 +1,8 @@
+.g-job-filter-container.clearfix
+  .timing
+  .checkbox
+    label
+      input.linear-scale(type="checkbox", checked=yScale=='linear')
+      | Use linear scale
+
+.g-jobs-graph
```

Comment:
Let's use a consistent naming convention in our pug files, the rest of them start with lowercase.